metron-interface/metron-rest/src/main/java/org/apache/metron/rest/controller/ParserExtensionController.java

```diff
@@ -90,8 +91,8 @@
   @ApiOperation(value = "Retrieves all ParserExtensionConfigs from Zookeeper")
   @ApiResponse(message = "Returns all ParserExtensionConfigs", code = 200)
   @RequestMapping(method = RequestMethod.GET)
-  ResponseEntity<Map<String, ParserExtensionConfig>> findAll() throws RestException {
-    return new ResponseEntity<Map<String, ParserExtensionConfig>>(extensionService.getAllParserExtensions(), HttpStatus.OK);
+  ResponseEntity<List<ParserExtensionConfig>> findAll() throws RestException {
```

Comment:
What is the logic for changing this from a map to a list?

Reply:
I have another branch where I am doing work on extending the management ui to manage extensions.  I had a lot of trouble consuming the maps.  The map is the configurations with a field pulled out as the key, so making it a list made it work, similar to the getAll for parser configurations, and really didn't lose anything.

TL;DR
I am not good enough at angular to consume the maps correctly so I refactored.

Reply:
Makes sense.  Its cleaner this way.  (Even if the real reason is that you aren't good enough with Angular.  Ha)